README.md

```diff
@@ -13,10 +13,11 @@ Its API is designed for performance-critical applications. It uses value types a
 ## Usage
 
 ### Declare SwiftPM dependency with release tag
+
 Add this repository to the `Package.swift` manifest of your project:
 
 ```swift
-// swift-tools-version:5.2
+// swift-tools-version:5.3
```

Comment:
Is there a particular reason to update the swift-tools-version? Doesn’t 5.2 work anymore?

Reply:
Haha, of course. I was even the one who updated that package version. Thanks for catching it 👍